go/lang/security/audit/crypto/tls.yaml

```diff
@@ -6,7 +6,9 @@ rules:
     metadata:
       cwe: 'CWE-327: Use of a Broken or Risky Cryptographic Algorithm'
       owasp: 'A9: Using Components with Known Vulnerabilities'
-      source_rule_url: https://github.com/securego/gosec/blob/master/rules/tls.go
+      source-rule-url: https://github.com/securego/gosec/blob/master/rules/tls.go
```

Comment:
Thoughts on `source-rule-url` being listed as a reference since these rules serve as inspiration and offer further reading?

Reply:
Thoughts are: Let's get this PR in and if it's important later, we can make a ticket to change.  
We originally "standardized" on this as a key since we were porting rules over.
I figure we could do either of these later: 1) write a script to add these to references or 2) have code in the viz layer that adds these to references